packages/ra-ui-materialui/src/detail/Tab.js

```diff
@@ -58,8 +58,9 @@ const sanitizeRestProps = ({
  *     );
  *     export default App;
  */
-class Tab extends Component {
-    renderHeader = ({ className, label, icon, value, translate, ...rest }) => (
+const Tab = props => {
+    const translate = useTranslate();
+    const renderHeader = ({ className, label, icon, value, ...rest }) => (
```

Comment:
This function is parameters and variables in the closure. I think you should decide between a function without parameters (and dereferencing all props in the Tab component signature), or a full-fledged component (in which case no parameter can be defined outside, as you did with translate). I'd personally opt for the first option.

Reply:
Ok, Done.